change/office-ui-fabric-react-2019-08-12-10-50-27-v-mare-SwatchColorPicker-fluent-styles.json

```
@@ -0,0 +1,8 @@
+{
+  "comment": "SwatchColorPicker: Updates web fluent styles per toolkit",
```

another nit: we should try to avoid `fluent` word now that it's by default fluent.
```suggestion
  "comment": "SwatchColorPicker: updates styles to align latest design specifications.",
```